### ISSUE TITLE
Feature Request - Power usage monitoring

### ISSUE THREAD

**Issue by WarrenSchultz** [jneilliii/OctoPrint-WemoSwitch#11]
Thanks for all your work on these plugins!

I see that the TP-Link plugin has energy monitoring on the Octoprint main screen, but the Wemo does not seem to have the same functionality. Is it possible to add that?
 Thanks!

**Comment by jneilliii** [comment]
Yeah, that was a new thing I added to the TPLink as I was gifted an energy monitoring device.  I'd be glad to look into the wemo options.  Is there a specific wemo device model that includes energy monitoring? Sorry, I had developed the plugin with an emulated device and I'm not sure all the same features are available through the emulated device.

**Comment by WarrenSchultz** [comment]
No worries :)

This is the device I was gifted:
Wemo Insight
https://www.amazon.com/dp/B01DBXNYCS

If you can find a way to implement support, I can assist with any testing, let me know.

**Comment by jneilliii** [comment]
Yeah, so it looks like it should be possible, but a little more difficult than with TPLink. Based on the information [here](https://ouimeaux.readthedocs.io/en/latest/api.html#insight) I can make individual calls for each of the different data points but there is not any information included with the normal get state call, which only returns on or off. If I find some free time I'll see about this integration.
